### PR TITLE
Config fix globals

### PR DESCRIPTION
**Pull request by QianyangPeng**
1. added another sort: syntax Mut ::= ".Mut" | "const" | "var"
2. globals now moved into <mainStore>, and <moduleInst> now has a <globalAddrs> for global addresses.
3. "init_global" is moved to test.md because it is used only for test. Maybe "init_locals" should also be moved to test.md in the future.
4. definition for init_global now changed to "init_global" INDEX GADDR. The test code also changed accordingly.

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -10,6 +10,27 @@ module WASM-TEST
...
+           ( .Bag =>
+             <globalInst>
+               <gAddr> GADDR </gAddr>
+               ...
+             </globalInst>
+           )
+           ...
+         </globals>
+```
+
```
Comment (ehildenb) [inline test.md]: Move this closer to where the code for `#assertGlobal` is. In the same section.

You can give `Globals` their own section (and separate them from `Locals`).

File: tests/simple/variables.wast [hunk 2]
```diff
@@ -19,8 +19,8 @@ init_locals < i32 > 0 : < i32 > 0 : < i32 > 0 : .Stack
 
 ;; Test globals
 
-init_global 0 < i32 > 0
-init_global 1 < i32 > 0
+init_global 0 0
+init_global 1 1
```
Comment (ehildenb) [inline tests/simple/variables.wast]: I think this should be something like:

```
init_global 0 0
(global.set 0 (i32.const 0))
init_global 1 1
(global.set 1 (i32.const 0))
```

Comment (QianyangPeng) [reply tests/simple/variables.wast]: added tests, implemented folded version and passed local check.

File: wasm.md [hunk 3]
```diff
@@ -526,20 +533,29 @@ The `*_local` instructions are defined here.
 ### Globals
 
 ```k
-    syntax Instr ::= "init_global" Int Val
-                   | "(" "global.get" Int ")"
+    syntax Instr ::= "(" "global.get" Int ")"
                    | "(" "global.set" Int ")"
+                   | "(" "global.set" Int Instr ")"
  // -----------------------------------------
```
Comment (ehildenb) [inline wasm.md]: Line up all the `")"` and make the `-----` extend to the end of the line:

```
    syntax Instr ::= "(" "global.get" Int       ")"
                   | "(" "global.set" Int       ")"
                   | "(" "global.set" Int Instr ")"
 // -----------------------------------------------
```

Comment (QianyangPeng) [reply wasm.md]: fixed

**Pull request merged**